databse/it-ausleihe.sql

```diff
@@ -0,0 +1,152 @@
+-- phpMyAdmin SQL Dump
+-- version 4.7.4
+-- https://www.phpmyadmin.net/
+--
+-- Host: 127.0.0.1
+-- Erstellungszeit: 12. Feb 2019 um 10:38
+-- Server-Version: 10.1.28-MariaDB
+-- PHP-Version: 7.1.11
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET AUTOCOMMIT = 0;
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Datenbank: `it-ausleihe`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Tabellenstruktur für Tabelle `borrower`
+--
+
+CREATE TABLE `borrower` (
+  `borrowerID` int(11) NOT NULL,
+  `name` varchar(25) NOT NULL,
+  `surname` varchar(25) NOT NULL,
+  `class` varchar(15) NOT NULL,
+  `teacher` varchar(30) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- --------------------------------------------------------
+
+--
+-- Tabellenstruktur für Tabelle `cart`
+--
+
+CREATE TABLE `cart` (
+  `cartID` varchar(6) NOT NULL,
+  `deviceID` varchar(6) NOT NULL,
+  `name` varchar(50) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- --------------------------------------------------------
+
+--
+-- Tabellenstruktur für Tabelle `device`
+--
+
+CREATE TABLE `device` (
+  `deviceID` varchar(6) NOT NULL,
+  `name` varchar(40) NOT NULL,
+  `type` varchar(10) NOT NULL,
+  `brand` varchar(20) NOT NULL,
+  `qrCode` int(1) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- --------------------------------------------------------
+
+--
+-- Tabellenstruktur für Tabelle `loan`
+--
+
+CREATE TABLE `loan` (
+  `loanID` int(11) NOT NULL,
+  `deviceID` varchar(6) NOT NULL,
+  `cartID` varchar(6) NOT NULL,
+  `borrowerID` int(11) NOT NULL,
+  `startTime` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
+  `duration` int(2) NOT NULL,
+  `returnTime` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
+  `comment` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+--
+-- Indizes der exportierten Tabellen
+--
+
+--
+-- Indizes für die Tabelle `borrower`
+--
+ALTER TABLE `borrower`
+  ADD PRIMARY KEY (`borrowerID`);
+
+--
+-- Indizes für die Tabelle `cart`
+--
+ALTER TABLE `cart`
+  ADD PRIMARY KEY (`cartID`),
+  ADD KEY `deviceID` (`deviceID`);
+
+--
+-- Indizes für die Tabelle `device`
+--
+ALTER TABLE `device`
+  ADD PRIMARY KEY (`deviceID`);
+
+--
+-- Indizes für die Tabelle `loan`
+--
+ALTER TABLE `loan`
+  ADD PRIMARY KEY (`loanID`),
+  ADD KEY `deviceID` (`deviceID`),
+  ADD KEY `deviceID_2` (`deviceID`),
```

Comment:
Somehow there is a "deviceID" and a "deviceID_2".

I am firmly convinced that the "deviceID_2" is not needed.